src/events.test.js

```diff
@@ -0,0 +1,54 @@
+import { getConfig } from '@edx/frontend-platform';
+import { postEventToIframe, postEventOutlineToSidebar } from './events';
+
+describe('postEventToIframe', () => {
+  test('sends a message to the iframe', () => {
+    const iframe = {
+      contentWindow: {
+        postMessage: jest.fn(),
+      },
+    };
+    const message = 'event_sucess';
+    const messageTargets = ['http://localhost:2000'];
+
+    postEventToIframe(iframe, message, messageTargets);
+
+    expect(iframe.contentWindow.postMessage).toHaveBeenCalledWith(
+      { message },
+      messageTargets[0],
+    );
+  });
+
+  test('if iframe is null, does not call postMessage', () => {
+    const iframe = null;
+    const message = 'event_success';
+    const messageTargets = ['http://localhost:2000'];
+    const postMessageMock = jest.fn();
+
+    postEventToIframe(iframe, message, messageTargets);
+
+    expect(postMessageMock).not.toHaveBeenCalled();
+  });
+});
+
+describe('postEventOutlineToSidebar', () => {
```

Comment:
If this is a unit test, I think you can focus on checking that "postEventToIframe" is called with the right parameters. @anfbermudezme

Reply:
I think `postEventOutlineToSidebar` is redundant so it's better to use `postEventToIframe` directly in `CoursewareContainer`.